src/config/dark-sites.config

```diff
@@ -1105,9 +1020,10 @@ spacex.com
 spark.lucko.me
 speedrun.com
 speedtest.net
+spicetify.app
```

Comment:
Not always dark... has button to switch dark/light mode:
![obraz](https://github.com/user-attachments/assets/350d3693-35cc-4e87-9eec-0e4b84689afe)